app/services/follow_service.rb

```diff
@@ -5,6 +5,9 @@ class FollowService < BaseService
   include Payloadable
   include DomainControlHelper
 
+  class Error < StandardError; end
```

Comment:
Can we re-use some existing classes here with custom messages or i18n strings?

Reply:
Sure! Will look through the codebase and see if I can identify an appropriate class to use.

Reply:
See comment above regarding i18n strings.